CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 ## Unreleased
 * Added a period at the end of lines produced by the **generate-docs** command that state the tested version of the product.
 * Update `RN112` validation's docs reference link.
+* Added support to control the maximum file size and log rotation files count in the sdk logger.
+* Fixed an issue with wrong recommendation for log level deprecated arguments in the sdk logger.
+* Fixed an issue with wrong documentation for log level arguments in the sdk logger.
```

Comment:
see the end [here](https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs)